src/lib/gossip_net/libp2p.ml

```diff
@@ -244,7 +244,14 @@ module Make (Rpc_intf : Network_peer.Rpc_intf.Rpc_interface_intf) :
       with
       | Ok (Ok net2) -> (
           let open Mina_net2 in
-          let me = config.keypair in
+          (* Make an ephemeral keypair for this session TODO: persist in the config dir *)
```

Comment:
Let's attach an issue number here.

Reply:
I just reverted to what's in compatible, but created an issue https://github.com/MinaProtocol/mina/issues/15495 that I can link.

Reply:
Convenient comment then :)